src/Frontend/Core/Engine/Block/ModuleExtra.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Frontend\Core\Engine\Block;
+
+use Frontend\Core\Engine\TwigTemplate;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+interface ModuleExtra
```

Comment:
Isn't it best practice to suffice interfaces with `Interface`, so `ModuleExtra` would become `ModuleExtraInterface`